[debug] [2018-09-09T20:37:59.560Z] ----------------------------------------------------------------------
[debug] [2018-09-09T20:37:59.562Z] Command:       /usr/bin/node /usr/bin/firebase serve --only functions
[debug] [2018-09-09T20:37:59.562Z] CLI Version:   4.2.1
[debug] [2018-09-09T20:37:59.563Z] Platform:      linux
[debug] [2018-09-09T20:37:59.563Z] Node Version:  v8.11.4
[debug] [2018-09-09T20:37:59.563Z] Time:          Sun Sep 09 2018 16:37:59 GMT-0400 (EDT)
[debug] [2018-09-09T20:37:59.563Z] ----------------------------------------------------------------------
[debug] 
[debug] [2018-09-09T20:37:59.569Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2018-09-09T20:37:59.569Z] > authorizing via signed-in user
[debug] [2018-09-09T20:37:59.571Z] >>> HTTP REQUEST GET https://admin.firebase.com/v1/projects/con-tac-toe  
 
[debug] [2018-09-09T20:37:59.893Z] <<< HTTP RESPONSE 200
[debug] [2018-09-09T20:37:59.894Z] >>> HTTP REQUEST GET https://cloudresourcemanager.googleapis.com/v1/projects/con-tac-toe  
 
[debug] [2018-09-09T20:38:00.097Z] <<< HTTP RESPONSE 200
[info] 
[info] === Serving from '/home/titocal/Documents/school work/2018 Fall/Software Engineering/group projects/con-tac-toe'...
[info] 
[debug] [2018-09-09T20:38:00.503Z] Error: EACCES: permission denied, mkdir '/usr/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/logs'
    at Object.fs.mkdirSync (fs.js:885:18)
    at Object.assertLogsPath (/usr/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/emulator/logs.js:65:10)
    at new Controller (/usr/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/cli/controller.js:77:32)
    at FunctionsEmulator.start (/usr/lib/node_modules/firebase-tools/lib/functionsEmulator.js:113:21)
    at Object.start (/usr/lib/node_modules/firebase-tools/lib/serve/functions.js:9:29)
    at /usr/lib/node_modules/firebase-tools/lib/serve/index.js:17:21
    at arrayMap (/usr/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:639:23)
    at Function.map (/usr/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:9554:14)
    at _serve (/usr/lib/node_modules/firebase-tools/lib/serve/index.js:15:7)
    at Command._action (/usr/lib/node_modules/firebase-tools/commands/serve.js:48:12)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2018-09-09T20:38:33.048Z] ----------------------------------------------------------------------
[debug] [2018-09-09T20:38:33.058Z] Command:       /usr/bin/node /usr/bin/firebase serve --only functions
[debug] [2018-09-09T20:38:33.058Z] CLI Version:   4.2.1
[debug] [2018-09-09T20:38:33.058Z] Platform:      linux
[debug] [2018-09-09T20:38:33.058Z] Node Version:  v8.11.4
[debug] [2018-09-09T20:38:33.058Z] Time:          Sun Sep 09 2018 16:38:33 GMT-0400 (EDT)
[debug] [2018-09-09T20:38:33.058Z] ----------------------------------------------------------------------
[debug] 
[debug] [2018-09-09T20:38:33.065Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2018-09-09T20:38:33.065Z] > authorizing via signed-in user
[debug] [2018-09-09T20:38:33.067Z] >>> HTTP REQUEST GET https://admin.firebase.com/v1/projects/con-tac-toe  
 
[debug] [2018-09-09T20:38:33.516Z] <<< HTTP RESPONSE 200
[debug] [2018-09-09T20:38:33.517Z] >>> HTTP REQUEST GET https://cloudresourcemanager.googleapis.com/v1/projects/con-tac-toe  
 
[debug] [2018-09-09T20:38:33.851Z] <<< HTTP RESPONSE 200
[info] 
[info] === Serving from '/home/titocal/Documents/school work/2018 Fall/Software Engineering/group projects/con-tac-toe'...
[info] 
[debug] [2018-09-09T20:38:34.256Z] Error: EACCES: permission denied, mkdir '/usr/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/logs'
    at Object.fs.mkdirSync (fs.js:885:18)
    at Object.assertLogsPath (/usr/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/emulator/logs.js:65:10)
    at new Controller (/usr/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/cli/controller.js:77:32)
    at FunctionsEmulator.start (/usr/lib/node_modules/firebase-tools/lib/functionsEmulator.js:113:21)
    at Object.start (/usr/lib/node_modules/firebase-tools/lib/serve/functions.js:9:29)
    at /usr/lib/node_modules/firebase-tools/lib/serve/index.js:17:21
    at arrayMap (/usr/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:639:23)
    at Function.map (/usr/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:9554:14)
    at _serve (/usr/lib/node_modules/firebase-tools/lib/serve/index.js:15:7)
    at Command._action (/usr/lib/node_modules/firebase-tools/commands/serve.js:48:12)
[error] 
[error] Error: An unexpected error has occurred.
